homeassistant/components/unifi/switch.py

```diff
@@ -150,8 +337,8 @@ def async_create_entity(event: ItemEvent, obj_id: str) -> None:
 
         api_handler.subscribe(async_create_entity, ItemEvent.ADDED)
 
-    for unifi_loader in UNIFI_LOADERS:
-        async_load_entities(unifi_loader)
+    for description in ENTITY_DESCRIPTIONS:
+        async_load_entities(description)
 
 
 @callback
```

Comment:
unrelated to this PR: `add_poe_entities` might benefit from some refactoring in a future PR to avoid a linear search inside of a linear search, but I don't know how frequently its call to know if it matters

Reply:
Should only happen on start up, after that its every time a new object is created, which shouldn't be very often

Reply:
I think you would need an `set` of all the clients on a specific port to be able to skip the inner loop.

Its really a question of if its worth keeping a `set` up to date or if `add_poe_entities` is called infrequently enough that it would cost more to keep the `set` in sync assuming you can only use it there.

Reply:
But this all doesn't matter as you are about to delete this code in https://github.com/home-assistant/core/pull/81749 👍

Reply:
> But this all doesn't matter as you are about to delete this code in #81749 👍

Yes finally :D